src/sinks/webhdfs/integration_tests.rs

```diff
@@ -0,0 +1,125 @@
+use std::{
```

Comment:
Given how long I've seen HDFS to start locally, it may be worth waiting to run the tests until we know it's up and properly running.

Reply:
Fixed by add health check in docker compose file.